lara-typescript/src/interactive-api-client/hooks.ts

```diff
@@ -176,3 +176,39 @@ export const useReportItem = <InteractiveState, AuthoredState>({ metadata, handl
     return () => client.removeGetReportItemAnswerListener();
   }, []);
 };
+
+export const DefaultAccessibilitySettings: IAccessibilitySettings = {
+  fontSize: "normal",
+  fontSizeInPx: 16
+};
+
+export const useAccessibility = (props?: {updateHtmlFontSize?: boolean, addBodyClass?: boolean}) => {
```

Comment:
This `useAccessibility` hook is nice. As long as clients use this, it will make it possible for us to introduce the idea of dynamic accessibility settings. Where students can change the settings and have the components update. I realize that isn't supported by the current initMessage approach of passing them around, but the `useAccessibility` abstracts that from the client.